ci/pvs-report.sh

```diff
@@ -7,54 +8,58 @@ set -x
 shopt -s failglob
 shopt -s dotglob
 
-BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
-source "$BUILD_DIR/ci/common/common.sh"
-source "$BUILD_DIR/ci/common/dependencies.sh"
-source "$BUILD_DIR/ci/common/deps-repo.sh"
-source "$BUILD_DIR/ci/common/doc.sh"
-source "$BUILD_DIR/ci/common/neovim.sh"
-source "$BUILD_DIR/ci/common/html.sh"
-source "$BUILD_DIR/ci/common/badge.sh"
+readonly BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+source "${BUILD_DIR}/ci/common/common.sh"
+source "${BUILD_DIR}/ci/common/dependencies.sh"
+source "${BUILD_DIR}/ci/common/deps-repo.sh"
+source "${BUILD_DIR}/ci/common/doc.sh"
+source "${BUILD_DIR}/ci/common/neovim.sh"
+source "${BUILD_DIR}/ci/common/html.sh"
+source "${BUILD_DIR}/ci/common/badge.sh"
```

Comment:
we've been actually removing `{}` when the variable is quoted and there's no ambiguity. Don't want to go back and forth on this, let's leave it as it was. Shellcheck shouldn't be complaining about that..